skimage/_shared/utils.py

```diff
@@ -655,8 +656,8 @@ def _validate_interpolation_order(image_dtype, order):
     if image_dtype == bool and order != 0:
         raise ValueError(
             "Input image dtype is bool. Interpolation is not defined "
-             "with bool data type. Please set order to 0 or explicitly "
-             "cast input image to another data type.")
+            "with bool data type. Please set order to 0 or explicitely "
```

Comment:
typo

Reply:
It is actually spelled "explicitly" https://dictionary.cambridge.org/dictionary/english/explicitly

Reply:
Hey @thanushipeiris,
that's what @stefanv was referring to, here :slightly_smiling_face: It now reads "explicitely," somehow:
>             "with bool data type. Please set order to 0 or explicitely "

Could you help us fixing it?
Thanks for your dedication to this PR! It looks like it's coming to an end. :slightly_smiling_face:

Reply:
ah my bad, I've tried to fix this now! 

Also, I tried merging some of the changes in the scikit image main branch which got a bit messy and has now added two new files: `tools/precompute/mc_meta/LookUpTable.h` and `dev.py` which I should not be making changes too...